prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -155,6 +155,11 @@ elements:
   - name: standard.specimen_type
     csvField: Specimen_type_code
 
+  # The type of device used for the test.  eg, BinaxNOW COVID-19 Ag Card
+  - name: standard.equipment_model_id
+    csvField: Device_ID
+
```

Comment:
Mapping Device_ID to existing 'standard.equipment_model_id'.    It wasn't clear to me what SimpleReport should use for this field, so its just a vague string at the moment.   HHS "How to report Covid ELR" notes imply that this should map to Column "O" in the LIVD spreadsheet, but that column is empty for some common tests, eg Binax.